<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ContactDetails</class>
 <widget class="QWidget" name="ContactDetails" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>620</height>
   </rect>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone - Contact Details</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="4" column="0" >
    <widget class="QLabel" name="label_21" >
     <property name="text" >
      <string>Web / blog:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3" >
    <widget class="QGroupBox" name="wengoGeneralDetailsGroup" >
     <property name="layoutDirection" >
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title" >
      <string>General Details</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="0" column="0" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_3" >
          <property name="text" >
           <string>Wengo alias:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1" >
         <widget class="QDateEdit" name="birthDate" >
          <property name="displayFormat" >
           <string>dd MMMM yyyy</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QLineEdit" name="firstName" >
          <property name="maxLength" >
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_5" >
          <property name="text" >
           <string>First name:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_6" >
          <property name="text" >
           <string>Last name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="alias" />
        </item>
        <item row="4" column="0" >
         <widget class="QLabel" name="label_7" >
          <property name="text" >
           <string>Birth Date:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="lastName" >
          <property name="maxLength" >
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="groupComboBox" />
        </item>
       </layout>
      </item>
      <item row="0" column="1" >
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="3" column="1" >
         <widget class="QLineEdit" name="city" >
          <property name="maxLength" >
           <number>20</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" >
         <widget class="QLabel" name="label_12" >
          <property name="text" >
           <string>State:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="label_10" >
          <property name="text" >
           <string>Gender:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QComboBox" name="country" />
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="label_11" >
          <property name="text" >
           <string>Country / Region:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1" >
         <widget class="QComboBox" name="state" />
        </item>
        <item row="3" column="0" >
         <widget class="QLabel" name="label_8" >
          <property name="text" >
           <string>City:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QComboBox" name="gender" >
          <item>
           <property name="text" >
            <string>Unknown</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Male</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Female</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="1" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" >
    <widget class="QLabel" name="label_19" >
     <property name="text" >
      <string>Email:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" >
    <widget class="QLineEdit" name="web" />
   </item>
   <item row="1" column="0" colspan="3" >
    <widget class="QGroupBox" name="IMAccountGroup" >
     <property name="title" >
      <string>Instant Messaging Identities</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="2" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="addIMContactButton" >
        <property name="text" >
         <string>Add IM Contact</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0" colspan="3" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>608</width>
         <height>32</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="cancelButton" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="saveButton" >
       <property name="text" >
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="3" >
    <widget class="QGroupBox" name="personalDetailsGroup" >
     <property name="title" >
      <string>Phone Numbers</string>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="1" column="3" >
       <widget class="QLineEdit" name="workPhone" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_18" >
        <property name="text" >
         <string>Work:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QLineEdit" name="cellPhone" />
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="wengoPhone" />
      </item>
      <item row="0" column="0" >
       <widget class="QLabel" name="label_15" >
        <property name="text" >
         <string>Mobile:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label_16" >
        <property name="text" >
         <string>Wengo:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QLineEdit" name="homePhone" />
      </item>
      <item row="0" column="2" >
       <widget class="QLabel" name="label_17" >
        <property name="text" >
         <string>Home:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="3" column="2" >
    <widget class="QLabel" name="avatarPixmap" >
     <property name="minimumSize" >
      <size>
       <width>91</width>
       <height>91</height>
      </size>
     </property>
     <property name="maximumSize" >
      <size>
       <width>91</width>
       <height>91</height>
      </size>
     </property>
     <property name="palette" >
      <palette>
       <active>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>195</red>
         <green>212</green>
         <blue>191</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </active>
       <inactive>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>234</red>
         <green>229</green>
         <blue>220</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>195</red>
         <green>212</green>
         <blue>191</blue>
        </color>
        <color>
         <red>64</red>
         <green>64</green>
         <blue>64</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </inactive>
       <disabled>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>212</red>
         <green>208</green>
         <blue>200</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>233</red>
         <green>231</green>
         <blue>227</blue>
        </color>
        <color>
         <red>128</red>
         <green>128</green>
         <blue>128</blue>
        </color>
        <color>
         <red>141</red>
         <green>139</green>
         <blue>133</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>106</red>
         <green>104</green>
         <blue>100</blue>
        </color>
        <color>
         <red>195</red>
         <green>212</green>
         <blue>191</blue>
        </color>
        <color>
         <red>195</red>
         <green>212</green>
         <blue>191</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>0</blue>
        </color>
        <color>
         <red>10</red>
         <green>36</green>
         <blue>106</blue>
        </color>
        <color>
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
        <color>
         <red>0</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>255</red>
         <green>0</green>
         <blue>255</blue>
        </color>
        <color>
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </disabled>
      </palette>
     </property>
     <property name="toolTip" >
      <string>Click to Change Avatar</string>
     </property>
     <property name="alignment" >
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap" >
      <bool>true</bool>
     </property>
     <property name="buddy" >
      <cstring>ContactDetails</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" >
    <widget class="QLineEdit" name="email" />
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>alias</tabstop>
  <tabstop>firstName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>birthDate</tabstop>
  <tabstop>gender</tabstop>
  <tabstop>country</tabstop>
  <tabstop>state</tabstop>
  <tabstop>city</tabstop>
  <tabstop>cellPhone</tabstop>
  <tabstop>wengoPhone</tabstop>
  <tabstop>homePhone</tabstop>
  <tabstop>workPhone</tabstop>
  <tabstop>email</tabstop>
  <tabstop>web</tabstop>
  <tabstop>saveButton</tabstop>
  <tabstop>cancelButton</tabstop>
  <tabstop>addIMContactButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
